### ISSUE TITLE
TypeError: reactRedux.connectAdvanced is not a function

### ISSUE THREAD

**Issue by GiancarlosIO** [keajs/kea#42]
Hi, i want to use kea v0.23.2 in a existing redux-project, but i'm getting the next error:
![captura de pantalla de 2017-09-13 16-42-22](https://user-images.githubusercontent.com/17274950/30403683-751e4246-98a8-11e7-9e22-8dcd7d000697.png)
![captura de pantalla de 2017-09-13 16-26-11](https://user-images.githubusercontent.com/17274950/30403781-dc3c4ef0-98a8-11e7-8324-09ecfdf5f89f.png)

Then i update the kea package to v0.24.1 and now when i try to use 'connect' function  i'm getting this error:
![captura de pantalla de 2017-09-13 17-22-35](https://user-images.githubusercontent.com/17274950/30403712-90562cb8-98a8-11e7-8fba-fbdf88ae6b50.png)

![captura de pantalla de 2017-09-13 16-46-14](https://user-images.githubusercontent.com/17274950/30403770-cb63c9fa-98a8-11e7-8ac1-0b427b8bfa93.png)

Thanks for advanced.

**Comment by mariusandra** [comment]
Hey, what version of `react-redux` are you using?

Try upgrading it. The `connectAdvanced` function that we use and seems to be missing for you was only recently added.

**Comment by GiancarlosIO** [comment]
@mariusandra but before kea was working normally...so strange...anyway I just update the react-redux package and now it work!, thanks dude :)
btw, in my team we are trying to write less-code-redux so we think kea can help us with that, actually in "production" we have a little piece of our app with kea.

**Comment by mariusandra** [comment]
Yeah... from some version of Kea, `react-redux` needed to be at v5 or later.
Hey, I added a stricter peer dependency for react-redux in kea. I added a [line to package.json](https://github.com/keajs/kea/commit/33bcdf86ab2a7770b82754b9e4af9f5d6c3b3429) specifying this.

Glad that it's resolved! Closing the issue :)

Also, if you wish, feel free to add your project to the "[Who's using Kea?](https://github.com/keajs/kea/issues/35)" issue :)
